modal/sandbox.py

```diff
@@ -390,7 +405,7 @@ def returncode(self) -> Optional[int]:
 
     @staticmethod
     async def list(
-        *, app_id: Optional[str] = None, client: Optional[_Client] = None
+        *, app_id: Optional[str] = None, tags: Optional[Dict[str, str]] = None, client: Optional[_Client] = None
     ) -> AsyncGenerator["_Sandbox", None]:
         """List all sandboxes for the current environment or app ID (if specified). Returns an iterator over `Sandbox`
```

Comment:
Docstring can be updated to mention `tags`

Reply:
done